pkg/security/config/default.go

```diff
@@ -7,56 +7,109 @@ package config
 
 // DefaultPolicy holds the default runtime security agent rules
 var DefaultPolicy = `---
-version: 1.0.0
+version: 1.0.1
```

Comment:
In the Pull request description you said

> Package the default policy fetch from the DataDog/security-agent-policies repository

But here it looks like also add a default policy directly integrated to the security-agent binary.

Reply:
Yes, this is temporary. It was decided to use the policies from the `DataDog/security-agent-policies` but we still want to test the default policy loads fine. We'll add ci-jobs on `security-agent-policies` that will use the latest build of the agent and test. Does it make sense ?

Reply:
We use this for testing purpose. We will remove this once we will have test/CI job on the policies repo side